client/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:22-bullseye AS runner
+
+WORKDIR /app
+
+COPY ./client ./client
+COPY ./local-dist ./local-dist
+
+WORKDIR /app/client
+
+RUN npm ci
+
+RUN if [ -d "../local-dist/embedding-sdk" ]; then \
+      npm install file:../local-dist/embedding-sdk --install-links; \
+    else \
+      echo "Local embedding-sdk dist is not found in ../local-dist/embedding-sdk, skipping copy"; \
+    fi
```

Comment:
Would the workflow be to `cp` our locally-built `metabase/resources/embedding-sdk` to this `local-dist` directory to use them?

Reply:
when this repo is fetched to the `metabase` repo, we will clone both Metabse Jar and Embedding SDK Dist to those `local-dist` of the app. And then when `running docker compose up` it will be installed

Reply:
ah, I see! sounds good to me then